packages/web-app/package.json

```diff
@@ -43,8 +44,6 @@
     "@types/node": "^20.14.8",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
-    "@typescript-eslint/parser": "^5.62.0",
```

Comment:
These were moved to the top level package.json since that is where we are running the lint command from. This nested package doesn't care about these so they shouldn't live here